akka-stream/src/main/scala/akka/stream/impl/fusing/ActorGraphInterpreter.scala

```diff
@@ -505,7 +514,7 @@ private[stream] final class GraphInterpreterShell(
       _isTerminated = true
```

Comment:
Thanks. (I only added that part to not make it sound like a command, but will omit in the future, thanks for letting me know that they were not achieving the intended effect)